### PR TITLE
Change “Render style from Template?” to “Use theme style?” when creating form #13369

### PR DESCRIPTION
**Pull request by fakela**

### CODE REVIEW

File: docs/components/forms.rst [hunk 1]
```diff
@@ -38,7 +38,7 @@ The following fields are available:
 
 - **Kiosk mode** - If Yes, Mautic turns off tracking of Contacts created through the Form, so that the Form doesn't generate cookies or associate any IP address with the Contact record. Marketers may refer to this as 'data entry mode'. It's ideal for using at conferences or events where several Contacts may enter their information using the same device, as it prevents associating the activity on the device to Contacts.
 
-- **Render style** - If Yes, the Form displays with the styling from either the selected Mautic Theme or the Attributes tab of the Form Fields. When No, the Form adopts the styling of where it's embedded.
+- **Use theme style** - If Yes, the Form displays with the styling from either the selected Mautic Theme or the Attributes tab of the Form Fields. When No, the Form adopts the styling of where it's embedded.
```
Comment (RCheesley) [inline docs/components/forms.rst]:
```suggestion
- **Use Theme style** - If Yes, the Form displays with the styling from either the selected Mautic Theme or the Attributes tab of the Form Fields. When No, the Form adopts the styling of where it's embedded.
```

**Review by RCheesley — CHANGES_REQUESTED** [review]
Just a tiny Vale fix then GTG!

**Comment by fakela** [comment]
@RCheesley comment resolved

**Review by RCheesley — APPROVED** [review]

**Review by RCheesley — APPROVED** [review]
GTG! 🚀

**Pull request merged**